projects/eslint-config/plugins/liferay/tests/lib/rules/no-abbreviations.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const MultiTester = require('../../../../../scripts/MultiTester');
+const rule = require('../../../lib/rules/no-abbreviations');
+
+const parserOptions = {
+	parserOptions: {
+		ecmaVersion: 6,
+		sourceType: 'module',
+	},
+};
+
+const ruleTester = new MultiTester(parserOptions);
+
+ruleTester.run('no-abbreviations', rule, {
+	invalid: [
+		{
+			code: `
+				e.preventDefault();
+			`,
+			errors: [
+				{
+					message:
+						'Avoid e abbreviation, preferred alternatives are [event,error]',
+					type: 'Identifier',
+				},
+			],
+		},
+
+		{
+			code: `
+				const btn = document.getElementById('button');
+			`,
+			errors: [
+				{
+					message:
+						'Avoid btn abbreviation, preferred alternative is button',
+					type: 'Identifier',
+				},
+			],
+		},
+	],
+
+	valid: [],
```

Comment:
Let's add a sanity check for regressions here, even just something like `(error) => error.toString()`